autoscheduler/frontend/src/components/App/App.tsx

```diff
@@ -9,9 +10,15 @@ import LandingPage from '../LandingPage/LandingPage';
 import SchedulingPage from '../SchedulingPage/SchedulingPage';
 import UnknownRoutePage from '../UnknownRoutePage/UnknownRoutePage';
 
-const App: React.SFC = function App() {
+const App: React.FC = function App() {
+  React.useEffect(() => {
+    const takeFullHeight = (): void => { document.body.style.height = `${window.innerHeight}px`; };
+    window.onresize = takeFullHeight;
+    return (): void => { window.removeEventListener('resize', takeFullHeight); };
+  }, []);
+
   return (
-    <div className={styles.appContainer}>
+    <Div100vh className={styles.appContainer}>
```

Comment:
is div100vh necessary still? I remember it being mentioned that it didn't do anything but I could be misremembering

Reply:
I got rid of it and couldn't tell the difference, at least on my Android or Chrome DevTools. I think you're right because I never did fix the bug where you rotate your iPhone and it has the empty space at the bottom, and Div100vh was intended as a possible solution for that

Reply:
It's been a while but I think we decided it was fine as long as it looked good on iPhone 6 in portrait mode